login-workflow/example/src/App.tsx

```diff
@@ -36,6 +40,10 @@ export const AuthUIConfiguration: React.FC = (props) => {
             //         instructions: 'Enter your employment information below.',
             //     },
             // ]}
+            // registrationSuccessScreen={(registrationData: RegistrationData): JSX.Element =>
+            //     customRegistrationSuccessScreen(registrationData)
```

Comment:
This looks much cleaner now that these are in another file. A small suggestion would be to make them functional components instead of just functions:

```tsx
type CustomRegistrationProps = {
    registrationData: RegistrationData;
}
export const CustomRegistrationSuccessScreen: React.FC<CustomRegistrationProps> = (props) => (
    <>
        ...
    </>
)
```

```tsx
registrationSuccessScreen={(registrationData: RegistrationData) => <CustomRegistrationSuccessScreen registrationData={registrationData} />}
```